cvmfs/monitor.cc

```diff
@@ -36,7 +36,7 @@
 #include <string>
 #include <vector>
 
-#ifndef CVMFS_LIBCVMFS
+#if !(defined(CVMFS_LIBCVMFS) || defined(CVMFS_SWISSKNIFE))
```

Comment:
I think we should use that expression to define a new name, e.g.

    #if !(defined(CVMFS_LIBCVMFS) || defined(CVMFS_SWISSKNIFE))
    #define CVMFS_FUSE_MODULE
    #endif 

and subsequently only use the `CVMFS_FUSE_MODULE`.  Or perhaps even better, let the build system single out the fuse module rather than everything else but the fuse module.